src/user/types/CompleteWhoAmI.ts

```diff
@@ -0,0 +1,51 @@
+import { AbstractEntity } from '../../abstractEntity';
+import {
+  CompleteWhoAmICompany,
+  CompleteWhoAmICompanyPaginationData,
+} from './CompleteWhoAmICompany';
+import {
+  CompleteWhoAmIUser,
+  CompleteWhoAmIUserPaginationData,
+} from './CompleteWhoAmIUser';
+import { CompleteWhoAmIResponseData } from '../UserClient';
+
+export enum CompleteWhoAmIResponseFields {
+  COLUMN_COMPANY = 'company',
+  COLUMN_USER = 'user',
+}
+
+export type PaginationData = {
```

Comment:
```suggestion
export type CompleteWhoAmIType = {
```

Reply:
I removed this type to respect constraints of GetResult<>